.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+  - enhancement
+# Label to use when marking an issue as stale
+staleLabel: stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as **stale** because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
```

Comment:
I wonder if it's possible to use any sort of variable substitution in this message, to customize it with the actual parameters governing Stale's activities? The repo I first noticed it in had it posting a message something like,
> This issue has been automatically marked as **stale** because it has been more than 30 days since the last update. It will automatically be closed in 7 days if no further activity occurs. Thank you for your contributions.

...Which is a little less brusque and more informative to the users whose issues it's threatening to close. But I have no idea if those parameters were pulled from the config, or if they were simply hard-coded into the text.

Then again, the config is YAML — it supports things like references out of the box. It might be possible to assemble the message using references to the YAML parameter values. I'll play with it a bit.

Reply:
Huh, I guess you [really can't](https://stackoverflow.com/questions/25852464/dry-up-yaml-with-variables) — support for any sort of variable replacement would have to be implemented by the bot author, in whatever language they use to consume the configuration. They _might_ have implemented that support, but it's not a built-in language feature. And the bot's docs are... spare. (Though I suppose I could try to remember which repo it was where I originally saw Stale in action, and go look at their config.)

Reply:
Well, fortunately that wasn't too hard.

As I feared, they simply [hardcoded the timespan values into the message text](https://github.com/travis-ci/travis-ci/blob/master/.github/stale.yml).

Reply:
I suppose we can always hardcode a better message in the future too. Especially if any confusion appears around this.

Reply:
Confusion, even, I can deal with. "Vehement backlash and incoherent rage" is more my worry. Internet haters gotta internet-hate.